third_party/lua/README-luadebug.md

```diff
@@ -83,7 +89,26 @@ If you have used other CLI debuggers, debugger.lua shouldn't be surprising. I di
     w|where $linecount
     - print source code around the current line
 
-If you've never used a command line debugger before, start a nice warm cozy fire, run tutorial.lua, and open it up in your favorite editor so you can follow along.
+If you've never used a command line debugger before, start a nice warm cozy fire, run luadebug_tutorial.lua, and open it up in your favorite editor so you can follow along.
+
+Breakpoints
+-
+
+One could set breakpoint either using full syntax as:
+
+  break debug-target.lua:9
+
+Or using "short" syntax using only line numbers in a form `+NN` or `:NN`. In this
+case debugger will use currently active script fil as a basic for such breakpoint.
+
+  b +9
+
+Both those syntaxes mentioned above are equivalent and should behave identically.
+
+If breakpoint(s) saved then execution (command 'continue') will be slowed down a
+bit for calling of debugger line hook at each executed line. If there is no any
+single breakpoint saved then execution (via `continue`) will be at the native speed,
+without any slowdown.
```

Comment:
Expanded description here and above in usage text:

```diff
diff --git a/third_party/c-dt b/third_party/c-dt
--- a/third_party/c-dt
+++ b/third_party/c-dt
@@ -1 +1 @@
-Subproject commit cec6acebb54d9e73ea0b99c63898732abd7683a6
+Subproject commit cec6acebb54d9e73ea0b99c63898732abd7683a6-dirty
diff --git a/third_party/lua/README-luadebug.md b/third_party/lua/README-luadebug.md
index 1bb1f857a..4724a8093 100644
--- a/third_party/lua/README-luadebug.md
+++ b/third_party/lua/README-luadebug.md
@@ -56,10 +56,10 @@ Debugger Commands:
 
 If you have used other CLI debuggers, debugger.lua shouldn't be surprising. I didn't make a fancy parser, so the commands are just single letters. Since the debugger is pretty simple there are only a small handful of commands anwyay.
 
-    b|break|breakpoint|add_break|add_breakpoint $address
-    - set new breakpoints at module.lua+num
-    bd|bdelete|delete_break|delete_breakpoint $address
-    - delete breakpoints
+    b|break|breakpoint|add_break|add_breakpoint $location
+    - set new breakpoint at given location in a form module.lua+num
+    bd|bdelete|delete_break|delete_breakpoint $location|*
+    - delete given breakpoint or all breakpoints if *
     bl|blist|list_break|list_breakpoints
     - list breakpoints
     c|cont|continue
@@ -110,6 +110,19 @@ bit for calling of debugger line hook at each executed line. If there is no any
 single breakpoint saved then execution (via `continue`) will be at the native speed,
 without any slowdown.
 
+You could see all activated breakpoints via `bl` command:
+
+  blist
+
+Saved breakpoint could be deactivated via `bd` command:
+
+  bdelete +9
+
+If you need to remove all breakpoints and continue execution at full speed
+then there is special breakpoint location name `*` could be used:
+
+  bd *
+
 Debugger API
 -
 
diff --git a/third_party/lua/luadebug.lua b/third_party/lua/luadebug.lua
index dfd2c66c6..a454779e7 100644
--- a/third_party/lua/luadebug.lua
+++ b/third_party/lua/luadebug.lua
@@ -717,8 +717,8 @@ local function cmd_quit()
 end
 
 local commands_help = {
-    {'b.reak.point $address|add_break.point', 'set new breakpoints at module.lua+num', cmd_add_breakpoint},
-    {'bd.elete $address|delete_break.point', 'delete breakpoints',  cmd_remove_breakpoint},
+    {'b.reak.point $location|add_break.point', 'set new breakpoints at module.lua+num', cmd_add_breakpoint},
+    {'bd.elete $location|delete_break.point', 'delete breakpoints',  cmd_remove_breakpoint},
     {'bl.ist|list_break.points', 'list breakpoints', cmd_list_breakpoints},
     {'c.ont.inue', 'continue execution', cmd_continue},
     {'d.own', 'move down the stack by one frame',  cmd_down},
```